libs/api/domains/driving-license/src/lib/graphql/studentInformationResult.model.ts

```diff
@@ -0,0 +1,17 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+import { IsNotEmpty, IsString, IsOptional } from 'class-validator'
+
+@ObjectType()
+export class StudentInformation {
+  @Field()
+  @IsString()
+  @IsNotEmpty()
+  name!: string
+}
+
+@ObjectType()
+export class StudentInformationResult {
+  @Field(() => StudentInformation, { nullable: true })
+  @IsOptional()
+  student!: StudentInformation | null
```

Comment:
Are the class validators on the return type possibly a mistake? Or am I missing something? (repeat)